src/modelize/modelize_class.nit

```diff
@@ -274,7 +274,7 @@ redef class ModelBuilder
 				if mtype == null then continue # Skip because of error
 				if not mtype isa MClassType then
 					error(ntype, "Error: supertypes cannot be a formal type.")
-					return
+					continue
```

Comment:
Hum. This error seems in fact never fired because of an overpowerful abort.

1. can you fix the error path?
2. can you add a test file? eg. tests/base_inheritance_formal.nit
3. can you check that --keep-going (default in nitpick) does not crash?